File: fizzbuzz.cpp
#include <stdio.h>
#include <iostream>
#include <conio.h>
using namespace std;

int main(void)
{
 int i;
 cout<<"fizzbuzz"<<endl<<endl;
 for(i=0;i<=30;i++)
 {
                   cout<<"  "<<i;
                   if(i%3==0)
                   {
                             cout<<"fizz";
                             }
                   if(i%5==0)
                   {
                                  cout<<"buzz"<<"  ";
                                  }
}
             getch();
}

File: fizzbuzz.scala
println("fizzbuzz");
println("ingrese un numero: ");
val n=readInt();
println(n);
for (i<-1 to n){
   println(" " + i + " ");
   if(i%3==0){
      println(" fizz ");
   }
   if(i%5==0){
      println(" buzz ");
   }
   if(i%3==0 && i%5==0){
      println(" fizzbuzz ");
   }
}
